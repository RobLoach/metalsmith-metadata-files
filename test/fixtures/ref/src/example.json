{
  "greeting": "metadata-files://thegreeting.json",
  "something": {
    "else": "metadata-files://thegreeting.json"
  }
}
