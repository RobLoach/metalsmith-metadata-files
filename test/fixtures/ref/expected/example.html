<div class="hello">Hello, World!</div>

<div class="goodbye">Goodbye, World!</div>
